jdaviz/configs/imviz/plugins/links_control/links_control.vue

```diff
@@ -30,15 +41,87 @@
+            :show_if_single_entry="multiselect"
+            :hint="multiselect ? 'Select viewers to set options simultaneously' : 'Select the viewer to set orientation'"
+          />
+          <plugin-layer-select
+            :items="layer_items"
+            :selected.sync="layer_selected"
+            :multiselect=false
+            :show_if_single_entry="true"
+            :label="'Orientation in viewer'"
+            :hint="'Select the viewer orientation'"
+          />
+          <v-row>
+            <span style="line-height: 36px">Presets:</span>
+            <j-tooltip tooltipcontent="Default orientation">
+              <v-btn icon @click="select_default_orientation">
+                <v-icon>mdi-restore</v-icon>
+              </v-btn>
+            </j-tooltip>
+            <j-tooltip tooltipcontent="north up, east left">
+              <v-btn icon @click="set_north_up_east_left">
+                <img :src="icon_nuel" width="24" class="invert-if-dark" style="opacity: 0.65"/>
+              </v-btn>
+            </j-tooltip>
+            <j-tooltip tooltipcontent="north up, east right">
+              <v-btn icon @click="set_north_up_east_right">
+                <img :src="icon_nuer" width="24" class="invert-if-dark" style="opacity: 0.65"/>
+              </v-btn>
+            </j-tooltip>
+          </v-row>
+        </div>
+
+        <div style="grid-area: 1/1">
+        </div>
+        <div v-if="link_type_selected == 'WCS'">
+
+          <j-plugin-section-header>Add orientation options</j-plugin-section-header>
+
+              <plugin-auto-label
+                :value.sync="new_layer_label"
+                :default="new_layer_label_default"
+                :auto.sync="new_layer_label_auto"
+                label="Name for orientation option"
+                hint="Label for this new model component."
+              ></plugin-auto-label>
+              <v-row>
+              <v-text-field
+                v-model="rotation_angle"
+                label="Rotation angle"
+                hint="Degrees counterclockwise from default orientation"
+                persistent-hint
+              ></v-text-field>
+              </v-row>
+                <v-row>
+                  <v-switch
+                    label="Rotate on add"
+                    hint="Select this orientation when added"
+                    v-model="set_on_create"
+                    persistent-hint>
+                  </v-switch>
+                </v-row>
+                <v-row>
+                  <v-switch
+                    label="East increases left"
+                    hint="Use the East-left convention"
+                    v-model="east_left"
+                    persistent-hint>
+                  </v-switch>
+                </v-row>
+                <v-row justify="end">
+                  <v-btn color="primary" color="accent" text @click="create_new_orientation_from_data">Add option</v-btn>

```

Comment:
this could benefit from a tooltip (and careful consideration of the terminology... 
@haticekaratay had a suggestion, if I remember correctly)

Reply:
That's right, @haticekaratay suggested `Option` -> `Orientation`. Sound ok to you too?